Umidigi/A13_Pro_Max_5G/res/values/integers.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <integer name="config_screenBrightnessSettingMaximum">255</integer>
+    <integer name="config_screenBrightnessSettingMinimum">3</integer>
+	<integer name="config_defaultPeakRefreshRate">61</integer>
+    <integer name="config_defaultRefreshRate">61</integer>
```

Comment:
What?

Reply:
From the vendor overlay of framework-res
![image](https://user-images.githubusercontent.com/949967/214029750-f4c7b092-e030-4213-8490-a109b50b7385.png)